prow/README.md

```diff
@@ -18,7 +18,7 @@ not make any attempt to preserve backwards compatibility.
 * `cmd/horologium` starts periodic jobs when necessary.
 * `cmd/mkpj` creates `ProwJobs`.
 
-See also: [Life of a Prow Job](https://github.com/kubernetes/test-infra/blob/master/prow/architecture.md).
+See also: [Life of a Prow Job](https://github.com/kubernetes/test-infra/blob/master/prow/architecture.md) and [Getting Started](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md).
```

Comment:
Can we put Getting started as a top level section? Even if it is just a link to this doc

Reply:
Sure, done.